src/endpoints.js

```diff
@@ -0,0 +1,39 @@
+const endpoints = {
+  submit: 'http://localhost:3001/api/v1/users',
```

Comment:
As I mentioned in the `eb-api-post` PR, what do you think about renaming `endpoints.submit` to `endpoints.users`? I think it'd be super helpful to know which endpoint we're actually hitting since that information is being held in this `endpoints.js` file.